app/src/main/java/com/itssuryansh/taaveez/Add_New_Content.kt

```diff
@@ -38,6 +51,13 @@ class Add_New_Content : AppCompatActivity() {
         loadLocate()
         loadDayNight()
         super.onCreate(savedInstanceState)
+        selectedLabels = HashSet()
```

Comment:
Can you try to create a database table for the labels, and fetch the labels from there?